src/azure-cli/azure/cli/command_modules/cosmosdb/_help.py

```diff
@@ -204,3 +204,437 @@
     text: az cosmosdb keys list --name MyCosmosDBDatabaseAccount --resource-group MyResourceGroup --subscription MySubscription
     crafted: true
 """
+
+helps['cosmosdb sql'] = """
+type: group
+short-summary: Manage Azure Cosmos DB SQL resources.
+"""
+
+# SQL database
+helps['cosmosdb sql database'] = """
+type: group
+short-summary: Manage Azure Cosmos DB SQL databases.
+"""
+
+helps['cosmosdb sql database create'] = """
+type: command
+short-summary: Create an SQL database under an Azure Cosmos DB database account.
+"""
+
+helps['cosmosdb sql database list'] = """
+type: command
+short-summary: List the SQL databases under an Azure Cosmos DB database account.
+"""
+
+helps['cosmosdb sql database show'] = """
+type: command
+short-summary: Show the details/properties of a SQL database under an Azure Cosmos DB database account.
+"""
+
+helps['cosmosdb sql database delete'] = """
+type: command
+short-summary: Delete the SQL database under an Azure Cosmos DB database account.
+"""
+
+# SQL container
+helps['cosmosdb sql container'] = """
+type: group
+short-summary: Manage Azure Cosmos DB SQL containers.
+"""
+
+helps['cosmosdb sql container create'] = """
+type: command
+short-summary: Create an SQL container under an Azure Cosmos DB SQL database.
+examples:
+  - name: Create an Azure Cosmos DB SQL container.
+    text: az cosmosdb sql container create -g MyResourceGroup -a MyAccount -d MyDatabase -n MyContainer --part "/my/path" --idx @policy-file.json --ttl 1000 --throughput "700"
+    crafted: true
+"""
+
+helps['cosmosdb sql container update'] = """
+type: command
+short-summary: Update an SQL container under an Azure Cosmos DB SQL database.
+"""
+
+helps['cosmosdb sql container list'] = """
+type: command
+short-summary: List the SQL containers under an Azure Cosmos DB SQL database.
+"""
+
+helps['cosmosdb sql container show'] = """
+type: command
+short-summary: Show the details/properties of a SQL container under an Azure Cosmos DB SQL database.
+"""
+
+helps['cosmosdb sql container delete'] = """
+type: command
+short-summary: Delete the SQL container under an Azure Cosmos DB SQL database.
+"""
+
+# SQL database throughput
+helps['cosmosdb sql database throughput'] = """
+type: group
+short-summary: Manage Azure Cosmos DB SQL database throughput.
+"""
+
+helps['cosmosdb sql database throughput show'] = """
+type: command
+short-summary: Get the throughput of the SQL database.
+"""
+
+helps['cosmosdb sql database throughput update'] = """
+type: command
+short-summary: Update the throughput of the SQL database.
+"""
+
+# SQL container throughput
+helps['cosmosdb sql container throughput'] = """
+type: group
+short-summary: Manage Azure Cosmos DB SQL container throughput.
```

Comment:
Consider something like "Manage throughput of SQL database under a Azure Cosmos DB account."

Please change the description for all throughput.